src/main/java/seedu/ui/Ui.java

```diff
@@ -32,11 +32,16 @@ public void printIntroduction() {
                 + "  88     88          88  \"8a,   ,d88  \"8a,   ,d88  \"8b,   ,aa    88,    \n"
                 + "  88     88          88   `\"8bbdP\"Y8   `\"YbbdP\"Y8   `\"Ybbd8\"'    \"Y888  \n"
                 + "                                       aa,    ,88                       \n"
-                + "                                        \"Y8bbdP\"                        ";
-        System.out.println(logo);
-        System.out.println("Hello!");
-        System.out.println("What would you like to do?");
-        System.out.println(SEPARATOR_LINE);
+                + "                                        \"Y8bbdP\"                        \n";
+        String greeting = "Hello!\n"
+                + "What would you like to do?\n";
+
+        String toPrint = SEPARATOR_LINE
```

Comment:
Can you rename this variable to something more descriptive? Maybe `introMessage`?